gtd-timer/Attributes/UserExistExceptionFilterAttribute.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using System;
+
+using gtdtimer.Exceptions;
+
+namespace gtdtimer.Attributes
+{
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class)]
+    public class UserExistExceptionFilterAttribute : ExceptionFilterAttribute
+    {
+        public override void OnException(ExceptionContext context)
+        {
+            if (context.Exception is UserExistException)
+            {
+                context.HttpContext.Response.StatusCode = 400;
```

Comment:
Please remove magic number

Reply:
Replaced 400 to HttpStatusCode.BadRequest